contracts/HATVaults.sol

```diff
@@ -50,7 +50,7 @@ contract  HATVaults is Governable, HATMaster {
     using SafeMath  for uint256;
     using SafeERC20 for IERC20;
 
-    struct PendingApproval {
+    struct SubmittedClaim {
```

Comment:
what about just calling it `Claim`?

Reply:
i thought so too but there's already a struct called Claim